src/GitHub.InlineReviews/Views/CommentView.xaml

```diff
@@ -170,18 +192,38 @@
                     <Style TargetType="FrameworkElement">
                         <Setter Property="Visibility" Value="Collapsed"/>
                         <Style.Triggers>
-                            <DataTrigger Binding="{Binding EditState}" Value="Editing">
+                            <DataTrigger Binding="{Binding EditState}" Value="Creating">
                                 <Setter Property="Visibility" Value="Visible"/>
                             </DataTrigger>
                         </Style.Triggers>
                     </Style>
                 </StackPanel.Style>
-                <Button Command="{Binding CommitEdit}" Content="{Binding CommitCaption}"/>
+                <Button Command="{Binding CommitCreate}" Content="{Binding CommitCaption}"/>
                 <Button Margin="4 0 0 0"
                         Command="{Binding StartReview}"
                         Visibility="{Binding CanStartReview, Converter={ui:BooleanToVisibilityConverter}}">
                     Start a review
                 </Button>
+                <Button Margin="4 0 0 0" Command="{Binding CancelCreate}">Cancel</Button>
+            </StackPanel>
+
+            <StackPanel Name="editingButtonPanel"
+                        Grid.Column="1" Grid.Row="3" 
+                        Margin="4 8" 
+                        HorizontalAlignment="Left"
+                        Orientation="Horizontal"
+                        IsVisibleChanged="buttonPanel_IsVisibleChanged">
+                <StackPanel.Style>
+                    <Style TargetType="FrameworkElement">
+                        <Setter Property="Visibility" Value="Collapsed"/>
+                        <Style.Triggers>
+                            <DataTrigger Binding="{Binding EditState}" Value="Editing">
+                                <Setter Property="Visibility" Value="Visible"/>
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </StackPanel.Style>
+                <Button Command="{Binding CommitEdit}">Edit</Button>
```

Comment:
Per @jcansdale's comment, can we change this to "Update comment" so it's inline with the interaction on dotcom?

Reply:
Fixed up.